tc39_test.go

```diff
@@ -269,6 +269,10 @@ var (
 
 		// generators
 		"test/annexB/built-ins/RegExp/RegExp-control-escape-russian-letter.js": true,
+
+		// duplicate checks in Proxy
+		"test/built-ins/Proxy/ownKeys/return-duplicate-symbol-entries-throws.js": true,
+		"test/built-ins/Proxy/ownKeys/return-duplicate-entries-throws.js":        true,
```

Comment:
This should probably be fixed, but I didn't want to touch it in this PR, but wanted to enable more tests